### ISSUE TITLE
is it possible to detect nested backticks?

### ISSUE THREAD

**Issue by ralyodio** [eslint/eslint#17933]
### Rule details

i have a module that is failing because it was auto-generated with nexted backticks

### Related ECMAScript feature

template-literals

### What type of rule is this?

Suggests an alternate way of doing something

### Example code

```js
const foo = `bar `baz``;
```

### Why should this rule be in the core instead of a plugin?

it could be either, but core seems to be against general syntax.

### Participation

- [ ] I am willing to submit a pull request to implement this rule.

**Comment by snitin315** [comment]
@ralyodio The provided code has a syntax error, hence ESLint is unable to parse this code. Unfortunately, I don't think ESLint can help you here, hence closing the issue.
